### DISCUSSION TITLE
[DHCP] Mac address blacklist.

### DISCUSSION THREAD

**Discussion by aToO57** [AdguardTeam/AdGuardHome#4034] [Q&A]
Hello,

I would like to switch from pi-hole to AdGuardHome but I have some problems.

I need to prevent some devices from getting an IP address from the AdGuardHome dhcp.

Is there any way to do this with AdGuardHome? (Pi-Hole allows this by using "dhcp-host=00:11:32:F5:BD,ignore" in the dsnmasq.d configuration).

Why do I ask this? Because my ISP provided TV set-top boxes need to receive their configurations from my modem/router.

**Comment by ainar-g** [comment]
Hello!  That sounds reasonable.  I've made this into a feature request #4037.

**Reply by aToO57** [reply]
> Hello! That sounds reasonable. I've made this into a feature request #4037.

Thank you !
